src/LiteCore/src/LiteCore.Shared/Interop/C4Replicator.cs

```diff
@@ -32,35 +32,38 @@ namespace LiteCore.Interop
     internal unsafe delegate void C4ReplicatorStatusChangedCallback(C4Replicator* replicator,
             C4ReplicatorStatus replicatorState, void* context);
 
-    [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
-    [return: MarshalAs(UnmanagedType.U1)]
-    internal unsafe delegate bool C4ReplicatorValidationFunction(FLSlice docID,
-            FLDict* body, void* context);
-
     [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
     internal unsafe delegate void C4ReplicatorDocumentErrorCallback(C4Replicator* replicator,
             [MarshalAs(UnmanagedType.U1)]bool pushing, FLSlice docID, C4Error error, 
             [MarshalAs(UnmanagedType.U1)]bool transient, void* context);
 
     [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
     internal unsafe delegate void C4ReplicatorBlobProgressCallback(C4Replicator* replicator,
-        [MarshalAs(UnmanagedType.U1)] bool pushing, FLSlice docID, FLSlice docProperty,
-        C4BlobKey blobKey, ulong bytesComplete, ulong bytesTotal, C4Error error, void* context);
+        [MarshalAs(UnmanagedType.U1)]bool pushing, FLSlice docID, FLSlice docProperty, 
+        C4BlobKey blobKey, ulong bytesComplete, ulong bytesTotal, C4Error error, 
+        void* context);
+
+    [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
+    [return: MarshalAs(UnmanagedType.U1)]
+    internal unsafe delegate bool C4ReplicatorPushFilterFunction(FLSlice docID, FLDict* body,
+        void* context);
 
     [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
-    internal unsafe delegate void C4ReplicatorPushFilterFunction(FLSlice docID,
-        FLDict* body, void* context);
+    [return: MarshalAs(UnmanagedType.U1)]
+    internal unsafe delegate bool C4ReplicatorValidationFunction(FLSlice docID, FLDict* body, 
+        void* context);
```

Comment:
These probably don't require different delegates since the signatures are the same.  They are both "Validation functions."